consensus/babe/babe.go

```diff
@@ -97,15 +102,12 @@ func (b *Session) invokeBlockAuthoring() {
 	var currentSlot uint64 = 0
 
 	for ; currentSlot < b.config.EpochLength; currentSlot++ {
-		if b.isProducer[currentSlot] {
-			// TODO: implement build block
-			block, err := b.buildBlock(big.NewInt(int64(currentSlot)))
-			if err != nil {
-				return
-			}
-			b.newBlocks <- *block
```

Comment:
can we call `buildBlock` with placeholders and keep `b.newBlocks <- block`?

Reply:
we still have `b.newBlock <- block` but block is a placeholder block now

Reply:
ah, I see :+1: